pam_vicon_o80/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Interface for the PAM Vicon system."""
+from .pam_vicon import PamVicon, NoFrameDataError
+
+
+__all__ = ("PamVicon", "NoFrameDataError")
```

Comment:
cool, I did not know about `__all__`

Reply:
I'm actually a bit unsure how useful it is.  Since `from foo import *` should not be used anyway (as it messes with linters), I only add it to suppress the "unused import" warnings.  It's a bit tedious to maintain, though, as all the imports need to be listed twice.  I recently found out that it's also possible to ignore specific warnings in specific files only, so I think just disabling the "unused import" warning for __init__.py is much easier (see for example the ruff config in pyproject.toml of this repo).